src/main/java/demo/guide_to_logback/Test.java

```diff
@@ -0,0 +1,24 @@
+package demo.guide_to_logback;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class Test {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Test.class);
```

Comment:
will this work correctly? if some of your dependencies was specified in pom with scope "test"?